koku/masu/processor/orchestrator.py

```diff
@@ -177,11 +186,11 @@ def start_manifest_processing(  # noqa: C901
         report_tasks = []
         for manifest in manifest_list:
             tracing_id = manifest.get("assembly_id", manifest.get("request_id", "no-request-id"))
-            files = manifest.get("files", [])
-            filenames = []
-            for file in files:
-                filenames.append(file.get("local_file"))
-            LOG.info(log_json(tracing_id, f"Report with manifest {tracing_id} contains the files: {filenames}"))
+            report_files = manifest.get("files", [])
+            filenames = [file.get("local_file") for file in report_files]
+            LOG.info(
```

Comment:
Logging inside loops often results in spamming the logger. It's usually better to log before/after the loop.